nilearn/plotting/displays.py

```diff
@@ -257,9 +257,15 @@ def transform_to_2d(self, data, affine):
                              self.direction)
         return cut
 
-    def draw_position(self, size, bg_color, **kwargs):
+    def draw_position(self, size, bg_color, decimals=False, **kwargs):
+        if decimals:
+            text = '%s=%.{}f'.format(decimals)
+            coord = float(self.coord)
```

Comment:
why is this necessary?

Reply:
I just wanted to keep original integer annotation and add a feature. but now it seems better to simplify. I will update the PR.